Documentation/Storage-Configuration/Ceph-CSI/ceph-csi-drivers.md

```diff
@@ -107,52 +117,52 @@ apiVersion: v1
-Also, See the example manifests for an [RBD ephemeral volume](https://github.com/rook/rook/tree/master/deploy/examples/csi/rbd/pod-ephemeral.yaml) and a [CephFS ephemeral volume](https://github.com/rook/rook/tree/master/deploy/examples/csi/cephfs/pod-ephemeral.yaml).
+Refer to the [ephemeral-doc](https://kubernetes.io/docs/concepts/storage/ephemeral-volumes/#generic-ephemeral-volumes) for more info.
+Example manifests for an [RBD ephemeral volume](https://github.com/rook/rook/tree/master/deploy/examples/csi/rbd/pod-ephemeral.yaml) and a [CephFS ephemeral volume](https://github.com/rook/rook/tree/master/deploy/examples/csi/cephfs/pod-ephemeral.yaml) on these pages.
 
 ## CSI-Addons Controller
 
-The CSI-Addons Controller handles the requests from users to initiate an operation. Users create a CR that the controller inspects, and forwards a request to one or more CSI-Addons side-cars for execution.
+The CSI-Addons Controller handles requests from users. Users create a CR
+that the controller inspects and forwards to one or more CSI-Addons sidecars for execution.
 
 ### Deploying the controller
 
-Users can deploy the controller by running the following commands:
+Administrators may deploy the controller by running the following commands:
 
 ```console
 kubectl create -f https://raw.githubusercontent.com/csi-addons/kubernetes-csi-addons/v0.7.0/deploy/controller/crds.yaml
 kubectl create -f https://raw.githubusercontent.com/csi-addons/kubernetes-csi-addons/v0.7.0/deploy/controller/rbac.yaml
 kubectl create -f https://raw.githubusercontent.com/csi-addons/kubernetes-csi-addons/v0.7.0/deploy/controller/setup-controller.yaml
 ```
 
-This creates the required crds and configure permissions.
+This creates the required CRDs and configures permissions.
 
 ### Enable the CSI-Addons Sidecar
 
 To use the features provided by the CSI-Addons, the `csi-addons`
 containers need to be deployed in the RBD provisioner and nodeplugin pods,
 which are not enabled by default.
 
-Execute the following command in the cluster to enable the CSI-Addons
-sidecar:
+Execute the following to enable the CSI-Addons sidecars:
 
 * Update the `rook-ceph-operator-config` configmap and patch the
- following configurations
+ following configuration:
 
 ```console
 kubectl patch cm rook-ceph-operator-config -nrook-ceph -p $'data:\n "CSI_ENABLE_CSIADDONS": "true"'
 ```
 
-* After enabling `CSI_ENABLE_CSIADDONS` in the configmap, a new sidecar container with name `csi-addons`
- should now start automatically in the RBD CSI provisioner and nodeplugin pods.
+* After enabling `CSI_ENABLE_CSIADDONS` in the configmap, a new sidecar container named `csi-addons`
+should now start automatically in the RBD CSI provisioner and nodeplugin pods.
 
 !!! note
-    Make sure the version of ceph-csi used is `v3.5.0+`.
+    The version of `ceph-csi` used must be at least `v3.5.0`.
```

Comment:
Let's remove this note completely. We are already beyond this min version of CSI.